bct/utils/visualization.py

```diff
@@ -269,7 +269,7 @@ def align_matrices(m1, m2, dfun='sqrdiff', verbose=False, H=1e6, Texp=1,
 
 @due.dcite(BibTeX(HIDALGO2007), "Weighted undirected network backbone")
 @due.dcite(BibTeX(HAGMANN2008), "Weighted undirected network backbone")
-def backbone_wu(CIJ, avgdeg):
+def backbone_wu(CIJ, avgdeg, verbose=False):
```

Comment:
I added verbosity as an option, so that bacbone_wu prints out the edges in the backbone (including the first edge, which was not the case before). I moved the print statement in the loop (see below line 322) a bit later as I believe the indexes of the original matrix make more sense.